bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -268,6 +329,61 @@ spec:
                   sleep 2
                   done
                 fi
+
+                election_timer="${OVN_SB_RAFT_ELECTION_TIMER}"
+                echo "Setting sb-db raft election timer to ${election_timer} ms"
+                retries=0
+                while current_election_timer=$(ovs-appctl -t /var/run/ovn/ovnsb_db.ctl cluster/status OVN_Southbound 2>/dev/null \
+                  | grep "Election" | sed "s/.*:[[:space:]]//"); do
```

Comment:
sometimes to be clear on non-greediness in matching up to a delimiter we can use a negated character class containing the delimiter

long way of asking if  `"s/[^:]*:[[:space:]]//"`  might be slightly better.

Reply:
@rbbratta I believe grep on constant string just before this sed makes it pretty less greedy ?

Reply:
if there were multiple `:` in the Election line the sed would match the second.   We usually try to avoid `.*` in general

but bikeshedding aside, you could use awk
```bash
| awk '/Election timer:/ {print $3}'
```
or grep   (using PCRE positive lookbehind assertion)
```bash
| grep -oP '(?<=Election timer:\s)[[:digit:]]+'
```

Reply:
Fixed.